File: PHP/1013 - O maior.php
<?php

    $numbers = (string)readline();
    $numbers = explode(" ", $numbers);

    $a = (int)$numbers[0];
    $b = (int)$numbers[1];
    $c = (int)$numbers[2];

    $maior_ab = ($a+$b+ abs($a-$b))/2;
    $maior_abc = ($maior_ab+$c+ abs($maior_ab-$c))/2;

    echo "$maior_abc eh o maior\n"
?>

File: CS/1013 - O maior.cs
using System;

class URI
{
    static void Main()
    {
        string[] numbers = Console.ReadLine().Split();

        int a = int.Parse(numbers[0]);
        int b = int.Parse(numbers[1]);
        int c = int.Parse(numbers[2]);

        int maior_ab = (a+b+Math.Abs(a-b))/2;
        int maior_abc = (maior_ab+c+Math.Abs(maior_ab-c))/2;

        Console.WriteLine(maior_abc + " eh o maior");
    }
}